backend/plugins/frontline_api/.env.sample

```
@@ -18,13 +18,9 @@ REDIS_PASSWORD=
 
-CALL_WS_API_USER=
-CALL_WS_API_PASSWORD=
-CALL_WS_SERVER=
-
 TIMEZONE=8
 
-CALL_CRON_API_USER=
-CALL_CRON_API_PASSWORD=
-
 ENDPOINT_URL=http://localhost:4444
+CALL_WEBHOOK_REQUIRE_SIGNATURE=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CALL_WEBHOOK_REQUIRE_SIGNATURE key should go before the ENDPOINT_URL key

(UnorderedKey)